cdc/model/kv.go

```diff
@@ -16,55 +16,40 @@ const (
 	OpTypeDelete OpType = 2
 )
 
-// RawKVEntry is entries received from TiKV
-type RawKVEntry = RegionFeedValue
-
-// KvOrResolved can be used as a union type of RawKVEntry and ResolvedSpan
-type KvOrResolved struct {
-	KV       *RawKVEntry
-	Resolved *ResolvedSpan
-}
-
-// ResolvedSpan represents that a span is resolved at a certain timestamp
-type ResolvedSpan struct {
-	Span      util.Span
-	Timestamp uint64
+// RegionFeedEvent from the kv layer.
+// Only one of the event will be setted.
+type RegionFeedEvent struct {
+	Val        *RawKVEntry
+	Checkpoint *ResolvedSpan
```

Comment:
why the name of this field is `Checkpoint`?

Reply:
I don't know why the name is chosen in the first place, it seems not a good name here.
But to avoid making the `diff` hard to review, I didn't rename it in this PR, maybe we can think of better names for these 3 structs involved later?

Reply:
it seems like the original name is `Resolved`, how about leave it as `Resolved`?

Reply:
Renamed.